packages/core/src/lib/tasks.ts

```diff
@@ -126,3 +126,13 @@ export async function installDeps(projectPath: string) {
     await execa.command(installCommand, { cwd: projectPath });
   }
 }
+
+export async function generateTypes(artefactsPath: string) {
+  try {
+    await execa.command(`npx typechain-polkadot --in . --out typedContract`, {
```

Comment:
This requires users to install typechain-polkadot beforehand?
If so, I think it's better to note that in README

Reply:
no, it is added to the dependency list, so npx should pick it up from there.